general-concepts/git/text.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<guide self="general-concepts/git/">
+<chapter>
+<title>Git for Gentoo Developers - General concepts</title>
+
+<section>
```

Comment:
I had some filler text here inside `<body>` and `<p>` but decided to take out as it didn't seem too important. Anything can be added afterwards obviously.